build.gradle

```diff
@@ -60,14 +59,18 @@ test {
 }
 
 task systemTest(type: Test, description: 'Runs the system tests.', group: 'Verification') {
+    // Retrieve properties passed in via -P that coincide with system tests. After "systemTest.", names should match those
+    // found in system-test-hub.properties
+    systemProperties project.getProperties().findAll { property ->
```

Comment:
Not sure how folks feel about passing in params to gradle tasks like this. I'm happy to use whatever's standard, though. This works, though (see https://ddt-jenkins.pdx.prod.flightstats.io/job/run-hub-system-tests/).

Reply:
what properties are passed here? Can you use system-test.properties file ?